src/pages/Login/Login.js

```diff
@@ -1,8 +1,91 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useNavigate, Link } from 'react-router-dom';
 import './Login.scss';
 
 const Login = () => {
-  return <div>Login</div>;
+  const [id, setId] = useState('');
+  const [pw, setPw] = useState('');
```

Comment:
```suggestion
  const [userInfo, setUserInfo] = useState({id:"",pw:""});
```
id state,
pw state를 
따로따로 관리하기보다, 객체로 한번에 관리 해 줄 수 있습니다!